File: WADV.CGModule/API/ConfigAPI.cs
using WADV.AppCore.API;

namespace WADV.CGModule.API
{
    /// <summary>
    /// 设置API类
    /// </summary>
    public static class ConfigAPI
    {

        /// <summary>
        /// 初始化CG模块
        /// </summary>
        /// <param name="dpi">图像默认DPI</param>
        /// <returns></returns>
        public static void Init(int dpi)
        {
            Config.DPI = dpi;
            Initialiser.LoadEffect();
            MessageAPI.SendSync("[CG]INIT_FINISH");
        }
    }
}

File: WADV.ShaderModule/API/ConfigAPI.vb
Namespace API
    Public NotInheritable Class ConfigAPI

        ''' <summary>
        ''' 初始化模块
        ''' </summary>
        ''' <remarks></remarks>
        Public Shared Sub Init()
            EffectList.ReadEffect()
            MessageAPI.SendSync("[SHADER]INIT_FINISH")
        End Sub
    End Class
End Namespace